pages/linux/rtcwake.md

```diff
@@ -0,0 +1,36 @@
+# rtcwake
+
+> Enter a system sleep state until specified wakeup time relative to your bios clock.
+> Commands need to be run as superuser since shutdowns are involved.
+
+- Check your hardware clock info and exit:
+
+`rtcwake -m show -v`
+
+- Show if an allarm is set:
+
+`rtcwake -m show`
+
+- Disable a previous set alarm:
+
+`rtc -m disable`
+
+- Suspend to ram and wakeup after 10 seconds:
+
+`# rtcwake -m mem -s {{10}}`
+
+- Suspend to disk _(higher power saving)_ and wakeup 10 minutes later:
```

Comment:
Not sure on italics and whether it renders correctly in the terminal. @agnivade, could you confirm?

Reply:
I don't think it usually does, at least not with a few of the clients. :thinking:

Reply:
It works fine on the node client. But usually we avoid markdown in the description, and this seems to be an incorrect usage of italicizing a text anyways. We should remove the underscores.